### ISSUE TITLE
Teleporting Mob's??

### ISSUE THREAD

**Issue by Painterman123** [Cazadorsniper/PO3#1210]
# Use the search function before reporting to see if the issue has been reported before.
**Pack Version:** 3.3.57
**Pack Mode (Normal, Titan, Kappa):** Normal
**Singleplayer, Multiplayer, or LAN?:** Single

**Description:** Mobs teleporting after hit. Not a Bug just want to turn this feature off.

**Steps to reproduce:**
Hit Mob
**Crash log (use pastebin or issue will be closed):**

**Comment by Cazadorsniper** [comment]
What mob?

**Comment by Painterman123** [comment]
All of them. My latest were 2 withers. They keep teleporting and are wreaking havoc.

**Comment by Cazadorsniper** [comment]
What is your tps when this happens?

**Comment by Painterman123** [comment]
If ticks per second I am not sure. Skellies will shoot at me then teleport away. I thought it might be the weapon I had but that is not the case. Strange but those damn Withers are what bothers me because they get to a certain low health then teleport somewhere else. Great Modpack though.
Breakout would be cool instead of Pong :)

**Comment by Cazadorsniper** [comment]
Did you add any mods?

**Comment by Painterman123** [comment]
Ok you got me. I did add Enchanting Plus. I could remove it and try it again.
The whole thing started when I killed 2 Minatour's but could not go into the fire swamp and read somewhere you had to eat one of drops like the meat stew or something. But like in a previous post my hunger bar would not go down so I could not eat. I copied my save files deleted the profile and re installed the Pack, It was a different version and all my Voidcraft stuff was gone so I started a new world with the updated Pack and noticed the mods teleporting. This probably doesn't help much. Sorry
